examples/src/main/java/io/kubernetes/client/examples/OutOfClusterClientExample.java

```diff
@@ -0,0 +1,45 @@
+/** Alipay.com Inc. Copyright (c) 2004-2019 All Rights Reserved. */
+package io.kubernetes.client.examples;
+
+import io.kubernetes.client.ApiClient;
+import io.kubernetes.client.ApiException;
+import io.kubernetes.client.Configuration;
+import io.kubernetes.client.apis.CoreV1Api;
+import io.kubernetes.client.models.V1Pod;
+import io.kubernetes.client.models.V1PodList;
+import io.kubernetes.client.util.ClientBuilder;
+import io.kubernetes.client.util.KubeConfig;
+import java.io.FileReader;
+import java.io.IOException;
+
+/**
+ * A simple example of how to use the Java API from an application outside a kubernetes cluster
+ *
+ * <p>Easiest way to run this: mvn exec:java
+ * -Dexec.mainClass="io.kubernetes.client.examples.OutOfClusterClientExample"
+ *
+ * <p>From inside $REPO_DIR/examples
+ */
+public class OutOfClusterClientExample {
```

Comment:
Perhaps `ConfigFileClusterExample` is a better name? Most people should just use the 'Config.defaultClient()' constructor which should work both inside a cluster and outside a cluster.

Reply:
yes, considering kubeconfig can be used either in cluster or out. renamed to `KubeConfigFileClientExample`?